viewing_party/party.py

```diff
@@ -0,0 +1,140 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = {
+        "title": movie_title,
+        "genre": genre,
+        "rating": rating
+    }
+
+    for key in new_movie:
+        if new_movie[key] == None:
+            return None
```

Comment:
Interesting approach of populating the possible return value, then iterating to check the values in a uniform fashion. In theory, this could reduce the chance of performing an incorrect check if the movie gained additional parameters. You'd only need to initialize the data value correctly.

Since we're really only looking at the values, and the project description indicates that we should return None if any of the inputs are falsy (None is a falsy value, but there are others), we might slightly modify this to something like:

```python
    for value in new_movie.values():
        if not value:
            return None
```